### PR TITLE
[stable/fluentd-elasticsearch] moved fluentd-elasticsearch from incubator to stable

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**: 
*  moved fluentd-elasticsearch from incubator to stable (as mentioned by @unguiculus incubator will die -> https://github.com/helm/charts/pull/6593#issuecomment-408612805)
* also added changes of https://github.com/helm/charts/pull/6707 & https://github.com/helm/charts/pull/6696
* changed configmap -> filters is matches now

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/fluentd-elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: A Fluentd Helm chart for Kubernetes with Elasticsearch output
+icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
+keywords:
+- fluentd
+- elasticsearch
+- multiline
+- detect-exceptions
+- logging
+sources:
+- https://github.com/kubernetes/charts/incubator/fluentd-elasticsearch
```
Comment (davidkarlsen) [inline stable/fluentd-elasticsearch/Chart.yaml]: old reference

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by monotek** [comment]
Thanks! Its also fixed in chart.yaml now.

**Pull request merged**